tests/conftest.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+
+
+@pytest.fixture(scope="session")
+def monkeysession(request):
```

Comment:
Nice, I had originally included a comment about consolidating this to prevent code duplication, as it already exists in two other places in the repo. Then I saw you did exactly that! Very nice.